cmake/Modules/opm-simulators-prereqs.cmake

```diff
@@ -22,10 +23,11 @@ set (opm-simulators_DEPS
 	"ERTPython"
         # OPM dependency
 	"opm-common REQUIRED;
-        opm-parser REQUIRED;
+   opm-parser REQUIRED;
```

Comment:
fixed, thanks. (be aware that this PR is not intended to be merged. at least not yet.)